homeassistant/components/huisbaasje/const.py

```diff
@@ -49,31 +50,62 @@
         "name": "Huisbaasje Current Power",
         "device_class": DEVICE_CLASS_POWER,
         "source_type": SOURCE_TYPE_ELECTRICITY,
-        "state_class": STATE_CLASS_MEASUREMENT,
     },
     {
-        "name": "Huisbaasje Current Power In",
+        "name": "Huisbaasje Current Power In Peak",
         "device_class": DEVICE_CLASS_POWER,
         "source_type": SOURCE_TYPE_ELECTRICITY_IN,
-        "state_class": STATE_CLASS_MEASUREMENT,
     },
     {
-        "name": "Huisbaasje Current Power In Low",
+        "name": "Huisbaasje Current Power In Off Peak",
         "device_class": DEVICE_CLASS_POWER,
         "source_type": SOURCE_TYPE_ELECTRICITY_IN_LOW,
-        "state_class": STATE_CLASS_MEASUREMENT,
     },
     {
-        "name": "Huisbaasje Current Power Out",
+        "name": "Huisbaasje Current Power Out Peak",
         "device_class": DEVICE_CLASS_POWER,
         "source_type": SOURCE_TYPE_ELECTRICITY_OUT,
-        "state_class": STATE_CLASS_MEASUREMENT,
     },
     {
-        "name": "Huisbaasje Current Power Out Low",
+        "name": "Huisbaasje Current Power Out Off Peak",
         "device_class": DEVICE_CLASS_POWER,
         "source_type": SOURCE_TYPE_ELECTRICITY_OUT_LOW,
-        "state_class": STATE_CLASS_MEASUREMENT,
+    },
+    {
+        "name": "Huisbaasje Energy Consumption Peak Today",
+        "device_class": DEVICE_CLASS_ENERGY,
+        "unit_of_measurement": ENERGY_KILO_WATT_HOUR,
+        "source_type": SOURCE_TYPE_ELECTRICITY_IN,
+        "sensor_type": SENSOR_TYPE_THIS_DAY,
+        "state_class": STATE_CLASS_TOTAL_INCREASING,
+        "precision": 1,
```

Comment:
Why add precision on 1 here?
Especially when using this with statistics and things like the energy dashboard, I think that might be a bit low as those stats are generated every hour? Maybe 2 or 3 is a better option?

Reply:
I think precision of 3 makes more sense as it boils down to watts then